Tests/Marketplace/search_and_install_packs.py

```diff
@@ -178,42 +179,48 @@ def install_packs(client, host, prints_manager, thread_index, packs_to_install,
         request_timeout (int): Timeout settings for the installation request.
     """
 
-    request_data = {
-        'packs': packs_to_install,
-        'ignoreWarnings': True
-    }
-
-    packs_to_install_str = ', '.join([pack['id'] for pack in packs_to_install])
-    message = 'Installing the following packs in server {}:\n{}'.format(host, packs_to_install_str)
-    prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN, include_timestamp=True)
-    prints_manager.execute_thread_prints(thread_index)
-
-    # make the pack installation request
-    try:
-        response_data, status_code, _ = demisto_client.generic_request_func(client,
-                                                                            path='/contentpacks/marketplace/install',
-                                                                            method='POST',
-                                                                            body=request_data,
-                                                                            accept='application/json',
-                                                                            _request_timeout=request_timeout)
+    if private_install:
+        local_packs = glob.glob("/home/runner/work/content-private/content-private/content/artifacts/packs/*.zip")
+        for local_pack in local_packs:
+            if any(pack_to_install['id'] in local_pack for pack_to_install in packs_to_install):
+                upload_zipped_packs(client=client, host=host, prints_manager=prints_manager, thread_index=thread_index, pack_path=local_pack)
+    else:
+        request_data = {
+            'packs': packs_to_install,
+            'ignoreWarnings': True
+        }
```

Comment:
why are we installing all the packs?

Reply:
It's just installing the packs which are being tested. There currently is an issue where the premium packs have to be installed manually due to limitations in setting the marketplace URL for premium

Reply:
Yeah but this line gets all of the content packs, no?
        local_packs = glob.glob("/home/runner/work/content-private/content-private/content/artifacts/packs/*.zip")

Reply:
That line grabs all of the packs which we have locally in the artifacts that we have available to upload. Then we loop through them and if they are in the list of packs_to_install then they get uploaded. If not it just continues the loop.